src/Randousel/randousel.svelte

```diff
@@ -0,0 +1,99 @@
+<script>
+  import { Card, CardSubtitle, CardTitle, CardText, CardActions, Button, MaterialApp } from 'svelte-materialify';
+  import { format } from 'd3';
+
+  export let location;
+  export let wtfData;
+  let count = 0;
+  let data = []; // Initialize data
+  let selectedIndex = 0; // Randomly pick an array element
+  let selectedWTF;
+
+  const loading = Promise.all([location, wtfData])
+    .then( ([locationResponse, wtfResponse]) => {
+      data = wtfResponse['data']['requests'].filter(d => d.wtf);
+      console.log(data)
+      selectedIndex = data.findIndex(d => d.state === locationResponse.region);
+    });
+
+  const amountFormatter = format('.2s');
+
+  function titleCase(str) {
+    str = str.toLowerCase().split(' ');
+    for (var i = 0; i < str.length; i++) {
+      str[i] = str[i].charAt(0).toUpperCase() + str[i].slice(1); 
+    }
+    return str.join(' ');
+  }
+
+  function randomize() {
```

Comment:
I don't think this is being used anywhere? If it does end up being used I may have further feedback. 🙃

Reply:
It isn't anymore & I should remove it. Would love further feedback tho.